docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,335 @@
+          Refer to <a href="#changes-in-141">Changes in 1.4.1</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.4.0</label>
+        </td>
+        <td>
+          Filters, regions, and teams are always referenced by its ID.
+          <br />
+          Disallowed <label>&lt;time&gt;</label> inside{" "}
+          <label>&lt;score&gt;</label> or <label>&lt;blitz&gt;</label> &
+          disallow <label>&lt;title&gt;</label> inside{" "}
+          <label>&lt;blitz&gt;</label>.
+          <br />
+          Required objectives.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.6</label>
+        </td>
+        <td>
+          Moved all defining elements out of module XML root.
+          <br />
+          Everything scores zero points by default.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.5</label>
+        </td>
+        <td>Filters is aware of who owns TNT.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.4</label>
+        </td>
+        <td>Wools have a location property.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.3</label>
+        </td>
+        <td>Define how overlapping regions should behave.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.2</label>
+        </td>
+        <td>Added monument modes.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.1</label>
+        </td>
+        <td>Off-by-one region bug fixed.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Map Protocol Changelog
+
+### Changes in 1.4.2
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- The objectives filter will now always automatically derive team ownership for the objective from context. To match an objective without any specific team, the <label>any="true"</label> attribute needs to be specified.
+  ***
+
+<span class="badge badge--success">New</span>
+<br />
+
+<span className="badge badge--secondary">Partially implemented</span>{" "}
+
+- Portals, kits, score boxes & structures can now be dynamically applied using dynamic filters. All region types are dynamic, however, only some filter types are dynamic.
+  - Kits now have [`give`, `take` & `lend` properties](/docs/modules/gear/kits#dynamic-kits).
+  - Portals have [`forward`, `reverse` & `transit` properties](/docs/modules/mechanics/portals#portal-sub-elements).
+  - Structures & [score boxes](/docs/modules/objectives/scoring#score-box-attributes) have a `trigger` property.
+- <span className="badge badge--secondary">Not implemented</span> Added a player
+  count filter.
+- Added a `<grounded/>` filter to check if the player is standing on the ground.
+- Added `<match-started/>`, `<match-running/>` & `<match-finished/>` filters that are used to check the state of the current match.
+- New `pre-match-physics` attribute for the terrain module to enable physics events before the match starts.
+
+### Changes in 1.4.1
```

Comment:
The source code doesn't mention a proto 1.4.1, these should be moved to 1.4.2

Reply:
I moved bullet 4, 5, and 7 in Proto 1.4.1 paragraph to be under 1.4.2 and removed everything else as it's only relevant for Minecraft 1.9 and newer.
![image](https://user-images.githubusercontent.com/20259871/193959388-de5c9a63-66bc-42f3-9a7f-b4e06648d4ab.png)

I also changed the Protocol Values table to show that 1.4.1 currently does nothing on PGM (from a quick testing of a map originally made with proto 1.4.0 changed to 1.4.1)
![image](https://user-images.githubusercontent.com/20259871/193959846-419cc942-7a50-42dc-be16-d6f6daf54bb0.png)

Reply:
Just saw the pic and want to mention: player count fikter *has* been reimplemented

Reply:
> Just saw the pic and want to mention: player count filter _has_ been reimplemented

Thanks for catching that. I must've missed the commit that re-implemented it and never got around to updating the page to reflect it.